OWNERS_ALIASES

```diff
@@ -0,0 +1,28 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+aliases:
```

Comment:
this is sort of needlessly opaque (and prevents emeritus-ing) if we're just going to use it in OWNERS/ anyhow?

Reply:
i can drop and we use just Owners